src/modules/AdvancedPaste/AdvancedPaste/Helpers/JsonHelper.cs

```diff
@@ -147,9 +150,9 @@ internal static string ToJsonFromXmlOrCsv(DataPackageView clipboardData)
                         }
 
                         // A CSV line is valid, if the delimiter occurs more or equal times in every line compared to the first data line. (More because sometimes the delimiter occurs in a data string.)
```

Comment:
```suggestion
                         // A CSV line is valid, if the delimiter occurs equal times in every line compared to the first data line
                         // and if every line contains no or an even count of quotation marks.
```

Reply:
Might be a commit mistake. But the second part regarding the quotation marks is missing.

Reply:
Resolved!